src/library/autodj/autodjprocessor.h

```diff
@@ -285,9 +285,11 @@ class AutoDJProcessor : public QObject {
     double m_transitionProgress;
     double m_transitionTime; // the desired value set by the user
     TransitionMode m_transitionMode;
+    bool m_crossfaderStartCenter;
 
     QList<DeckAttributes*> m_decks;
 
+    
     ControlProxy* m_pCOCrossfader;
```

Comment:
remove

Reply:
I don't think m_pCOCrossfader has anything to do with my code, other than I used it as a template for some of my changes.  I think m_pCOCrossfader refers to https://github.com/mixxxdj/mixxx/pull/13303.